src/utilities.js

```diff
@@ -159,6 +159,33 @@ export function getItemText(amount) {
     }
 }
 
+// Returns a rough estimate of the pixel length of
+// a string based on the letters in the string
+// in RS font. Didn't consider numbers or other symbols.
+export function getEstimatedPixelLength(str) {
+    let i = str.length;
+    let totalLength = 0;
+
+    while (i--) {
+        switch (true) {
+            case ["i", "l"].includes(str[i]):
+                totalLength += 1;
+                break;
+            case ["m", "w"].includes(str[i]):
+                totalLength += 8;
+                break;
+            case str[i] == str[i].toUpperCase():
+                totalLength += 8;
+                break;
+            default:
+                totalLength += 6;
+                break;
+        }
+    }
```

Comment:
Recently I've been liking the if return, if return... format over switch case in times like this where switch isn't actually used

```js
export const getEstimatedPixelWidth = (str) => 
    str.split("").reduce((sum, char) => {
        if (["i", "l"].includes(char)) return sum + 1;
        if (["m", "w"].includes(char)) return sum + 8;
        if (char == char.toUpperCase()) return sum + 8;
        return sum + 6;
    });
```